src/main/java/thestonedturtle/mahoganyhomes/TeleportItems.java

```diff
@@ -0,0 +1,186 @@
+package thestonedturtle.mahoganyhomes;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import net.runelite.api.Client;
+import net.runelite.api.InventoryID;
+import net.runelite.api.Item;
+import net.runelite.api.ItemID;
+import net.runelite.api.ItemContainer;
+
+@AllArgsConstructor
+public enum TeleportItems
+{
+	// East Ardy
+	JESS(Arrays.asList(
```

Comment:
No need for `Arrays.asList` as we do not need to store these as a list, a `TeleportItem[]` is good enough. (see other comment related to `Collections.sort()`)

So, lets remove `@AllArgsConstructor`, unwrap all of these,  and then add this constructor & update the `teleportItems` type:

```java
	private final TeleportItem[] teleportItems;
	
	TeleportItems(TeleportItem... items) {
		this.teleportItems = items;
	}
```

Side note: you probably wanted to use `List.of` instead of `Arrays.asList` since the method you used returns a muttable array where `List.of` would return immutable and this array never changes. RuneLite supports java features up to version 11 now